package.json

```diff
@@ -143,11 +143,13 @@
     "zen-observable-ts": "0.8.10"
   },
   "dependencies": {
+    "@types/memoize-one": "^3.1.1",
```

Comment:
Should be a dev dep I guess

Reply:
good catch! fixed

Reply:
no problem, thanks fot the work you‘re doin‘ for open source 🙂